Tests/MapboxMapsTests/Foundation/Mocks/MockStyleManager.swift

```diff
@@ -0,0 +1,343 @@
+import Foundation
+@testable import MapboxMaps
+@_implementationOnly import MapboxCommon_Private
+
+final class MockStyleManager: StyleManagerType, Stubbable {
+
+    func asStyleManager() -> StyleManager {
+        fatalError()
+    }
+
+    @Stubbed var stubStyleURI: String!
+    func getStyleURI() -> String {
+        stubStyleURI ?? ""
+    }
+
+    func setStyleURIForUri(_ uri: String) {
+        stubStyleURI = uri
+    }
+
+    @Stubbed var stubStyleJSON: String!
```

Comment:
```suggestion
    @Stubbed var stubStyleJSON: String!
```

In other mocks, we follow a consistent naming convention for stub properties: `styleJSONStub` would be more consistent.